packages/editor/src/components/widgets/table/row/SelectRow.tsx

```diff
@@ -6,13 +6,24 @@ type SelectRowProps<TData> = {
   row: Row<TData>;
   children: ReactNode;
   isNotSelectable?: boolean;
+  onDoubleClick?: () => void;
 };
 
-export const SelectRow = <TData extends object>({ row, children, isNotSelectable }: SelectRowProps<TData>) => (
+export const SelectRow = <TData extends object>({ row, children, isNotSelectable, onDoubleClick }: SelectRowProps<TData>) => (
   <tr
     className={isNotSelectable ? '' : 'selectable-row'}
     data-state={row.getIsSelected() ? 'selected' : ''}
-    onClick={isNotSelectable ? undefined : row.getToggleSelectedHandler()}
+    onClick={event => {
+      if (event.detail === 1) {
+        if (!isNotSelectable) {
+          row.getToggleSelectedHandler()(event);
+        }
+      } else if (onDoubleClick) {
+        if (!isNotSelectable) {
+          onDoubleClick();
+        }
+      }
```

Comment:
move isnotselectable to the start, instead of check it in each if block

Reply:
👍